### ISSUE TITLE
How to Reset the "Color and Themes" Section?

### ISSUE THREAD

**Issue by Ryonez** [causefx/Organizr#1085]
###### Operating System: docker
<hr>

##### Problem Description:

How do we reset the "Color and Themes" section in the settings? Using the placeholder hex doesn't revert changes.
<hr>

##### Reproduction Steps:

Change the color hex codes in "Color and Themes".
Then try to revert the changes.
<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by tronyx** [comment]
Should be able to just put the placeholder color code back in, #987654, or select one of the preset themes.

**Comment by causefx** [comment]
just delete the field and hit save

**Comment by Ryonez** [comment]
That did the trick causefx. Could you include that info with the placeholder info for others please? That really didn't seem intuitive.

**Comment by tronyx** [comment]
@causefx Do we want to maybe add a reset button to make it a little easier?

**Comment by causefx** [comment]
yea. i will look into it. it’s the shitty js framework :/

**Comment by hirenshah** [comment]
I came across this logged issue when entering #987654 in the field with the hope of resetting it just got reverted after saving and refreshing.  Saving it with a blank entry does the trick but as @Ryonez mentioned, it isn't something that is obvious.

@christronyxyocum's suggestion would be ideal, however on the scale of things, this "issue" is fairly minor.  @causefx, a temporary workaround would be (as @Ryonez mentioned) to just add to the end of the following string:

> The value of #987654 is just a placeholder, you can change to any value you like.

Something like:

> To revert back to default, save with no value defined in the relevant field.
